src/brasil/gov/paginadestaque/tests/test_setup.py

```diff
@@ -77,12 +79,78 @@ def test_tile_enabled(self):
         available_tiles = api.portal.get_registry_record(record)
         self.assertIn(u'spacer', available_tiles)
 
+
+class TestUpgrade(BaseTestCase):
+    """Ensure product upgrades work."""
+
+    def list_upgrades(self, source, destination):
+        upgradeSteps = listUpgradeSteps(self.st,
```

Comment:
follow conventions for [breaking lines](http://docs.plone.org/external/plone.api/docs/contribute/conventions.html#breaking-lines) here and all over the code